backend/app/services/ml_engine.py

```
@@ -0,0 +1,376 @@
+    "Ascending Aorta": "Восходящая аорта",
+    "max_diam_1": "Максимальные диаметры",
+    "max_diam_2": "Максимальные диаметры",
+    "min_diam": "Минимальный диаметр",
+    "perimetr": "Периметр сосуда",
+    "area": "Площадь поперечного сечения"
+}
+
+def _normalize_text(text: str) -> set:
+    import re, pymorphy3
+    morph = pymorphy3.MorphAnalyzer()
+    words = re.findall(r'\b[а-яА-ЯёЁa-zA-Z]{3,}\b', text.lower())
```

Ruff (0.15.12)

[warning] String contains ambiguous `а` (CYRILLIC SMALL LETTER A). Did you mean `a` (LATIN SMALL LETTER A)?

(RUF001)